In [4]:
import pandas as pd
data = pd.read_csv('FoodPrice_in_Turkey.csv', header = 0, encoding = "ISO - 8859-1") # Không khai báo gì thì encoding là Unicode
data.head()

Place  ProductId    ProductName  UmId UmName  Month  Year  \
0  National Average         52  Rice - Retail     5     KG      5  2013   
1  National Average         52  Rice - Retail     5     KG      6  2013   
2  National Average         52  Rice - Retail     5     KG     11  2013   
3  National Average         52  Rice - Retail     5     KG     12  2013   
4  National Average         52  Rice - Retail     5     KG      1  2014   

    Price  
0  4.4920  
1  4.5786  
2  4.7865  
3  5.1337  
4  5.5099

### Kiểm tra thông tin

In [5]:
data.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 7381 entries, 0 to 7380
Data columns (total 8 columns):
 #   Column       Non-Null Count  Dtype  
---  ------       --------------  -----  
 0   Place        7381 non-null   object 
 1   ProductId    7381 non-null   int64  
 2   ProductName  7381 non-null   object 
 3   UmId         7381 non-null   int64  
 4   UmName       7381 non-null   object 
 5   Month        7381 non-null   int64  
 6   Year         7381 non-null   int64  
 7   Price        7381 non-null   float64
dtypes: float64(1), int64(4), object(3)
memory usage: 461.4+ KB


## Giải đáp thắc mắc

### 1. Bộ dữ liệu chứa bao nhiêu dòng, bao nhiêu cột

Bộ dữ liệu có 8 cột, 7381 dòng

### 2. Thang đo tương ứng của các thuộc tính được lưu trữ

Thang đo định danh: Place, ProductName, UmName, ProductID, UmID
Thang đo tỉ lệ: Month, Year, Price

### 3. Giá trung bình của từng loại thực phẩm là bao nhiêu

In [15]:
Ave_price = data.groupby('ProductName')['Price'].mean()
Ave_price.head()

ProductName
Apples (red) - Retail      2.392193
Apples - Retail            3.885039
Bananas - Retail           8.224784
Beans (white) - Retail     9.572830
Bread (common) - Retail    4.463293
Name: Price, dtype: float64

### 4. Đề xuất những loại phân tích mà bạn muốn biết mà bộ dữ liệu này có thể cung cấp

In [ ]:
+ Top 10 loại thực phẩm giá cao nhất
+ Top 10 loại thực phhẩm giá thấp nhất